plugins/inputs/ethtool/ethtool_test.go

```diff
@@ -1,5 +1,3 @@
-//go:build linux
```

Comment:
I'm not seeing a good reason to run tests on non-Linux platforms when only Linux has ethtool.

Reply:
I develop primarily on macOS. Since the tests use mock interfaces (and namespaces) it was helpful to be able to test filter logic locally.

Reply:
Note that prior to the abstraction layer for namespaces, the use of ethtool was less abstracted, so unit tests couldn't be run on Linux because the code would not compile. Adding the namespaces necessitated and increase in abstraction, or for unit tests to be run as root.